platform/security/sts/security-sts-propertyclaimshandler/src/test/java/org/codice/ddf/security/sts/claims/property/TestAttributeFileClaimsHandler.java

```diff
@@ -158,6 +160,42 @@ public void testRetrieveClaimsValuesNullPrincipal() {
     ProcessedClaimCollection processedClaims =
         attributeFileClaimsHandler.retrieveClaimValues(claimCollection, claimsParameters);
 
-    Assert.assertThat(processedClaims.size(), CoreMatchers.is(equalTo(0)));
+    assertThat(processedClaims, is(empty()));
+  }
+
+  @Test
+  public void testGetSingleValuedSystemHighAttribute() throws IOException {
+    assertThat(attributeFileClaimsHandler.getValues("test"), containsInAnyOrder("testValue"));
+  }
+
+  @Test
+  public void testGetMultiValuedSystemHighAttribute() throws IOException {
+    assertThat(
+        attributeFileClaimsHandler.getValues("test1"),
+        containsInAnyOrder("testing1", "testing2", "testing3"));
+  }
+
+  @Test
+  public void testNoMatchGetSystemHighAttribute() throws IOException {
+    assertThat(attributeFileClaimsHandler.getValues("reg"), is(empty()));
+  }
+
+  @Test(expected = Exception.class)
```

Comment:
❓  Are they actually throwing an `Exception` instance? If not we should consider expecting the specific exception being thrown, unless the thrower of the exception is a third party. 

This comment also applies to the below uses of `@Test(expected = Exception.class)` as well.

Reply:
❗️ Agreed. We should use specific exceptions, not just `Exception`. Comment applies to the following tests as well.

Reply:
I don't know completely what the DDF convention is, but `Exception` makes more sense to me. This unit test is verifying that the initialization of `AttributeFileClaimsHandler` will fail if the `users.atrtibutes` file doesn't exist. We know when bundles fail to initialize when we go through the installer. I don't care about asserting exactly what kind of exception but rather that there is an exception which indicates that the initialization to fail. That is, if the implementation changed later to throw a different exception when the `users.attribtues` file doesn't exist, this unit test will still pass.